data/korath/korath missions.txt

```diff
@@ -257,3 +257,81 @@ mission "Small Scale Delivery"
 	on complete
 		payment 1000
 		dialog `You find a Korath who seems to have been waiting for you in the spaceport. They greet you by showing their palms, then outstretch their hands as if waiting to receive the package. You hand off the small delivery and they hand you a small payment of <payment>.`
+
+mission "Efret: Quarg Rescue"
+	boarding
+	repeat
+	invisible
+	deadline 1
+	source
+		government "Kor Efret"
+	on offer
+		"old quarg rep" = "reputation: Quarg"
+		# bring it low enough that capping this will result in war
+		"reputation: Quarg" <?= 1
+		conversation
+			branch "unfriendly quarg" # if you're already at war with them
+				"reputation: Quarg" < 0
+			`The Quarg have brought reinforcements, probably in reaction to a distress call by the <ship>. They hail you, with barely contained anger in their voice. "Remove yourself from hence, and we may yet forgive your aggression. The Kor Efret are under our protection! Think not even about capturing yonder vessel."`
```

Comment:
I assume this starts this way to make it shocking or intimidating? There's usually some small preamble setting the scene of a boarding mission.
I'm not sure this achieves what I'm assuming was the intent. It just seems kind of weird and disjointed.

Reply:
Yeah I probably did not take enough time to write some parts properly
Should prob add you were about to board it or smth